cypress/integration/supervision/letters/create-letter.spec.js

```diff
@@ -44,33 +36,43 @@ describe('Creating, editing and retrieving a letter', { tags: ['@supervision', '
     'Given I\'m a Case Manager on Supervision creating a letter template' +
     'Then the letter is edited as expected',
     () => {
-      cy.get('.title-person-name', {timeout: 30000}).contains('Ted Tedson');
+      cy.get('.title-person-name').contains('Ted Tedson');
 
       cy.get('@orderId').then(orderId => {
         cy.get('@clientId').then(clientId => {
-          cy.visit('/supervision/#/clients/' + clientId + '/orders/' + orderId + '/drafts/create/template');
+          cy.visit(`/supervision/#/clients/${clientId}/orders/${orderId}/drafts/create/template`);
         });
       });
 
-      cy.get('.select-template', {timeout: 30000}).contains('Select a template');
+      cy.contains('.select-template', 'Select a template')
       cy.contains('blank: Blank template').click();
 
-      getRecipientHeader().contains('Select recipient/s');
-      getFirstSelectableRecipient().click();
+      cy.get('.smart__action > ng-component:nth-child(2) > div:nth-child(1) > draft-select-recipients:nth-child(1) > div:nth-child(1) > div:nth-child(1) > h2:nth-child(1)')
```

Comment:
This is a pretty big selector. Is there any nicer way to target the recipient header element?